cli/src/main/java/com/devonfw/tools/ide/tool/ToolCommandlet.java

```diff
@@ -270,7 +270,13 @@ protected void extract(Path file, Path targetDir) {
       fileAccess.delete(tmpDir);
     } else {
       this.context.trace("Extraction is disabled for '{}' hence just moving the downloaded file {}.", getName(), file);
-      fileAccess.move(file, targetDir);
+
+      try {
+        Files.createDirectories(targetDir);
+      } catch (IOException e) {
+        throw new IllegalStateException("Failed to create folder " + targetDir);
+      }
+      fileAccess.move(file, targetDir.resolve(file.getFileName()));
```

Comment:
Where do you see the additional folders situation? Because currently the software tools are organized in `_ide/software/default/eclipse/eclipse` anyway or have we already changed that?

I installed `eclipse` with the the unchanged behaviour and there was no difference in the folder structure. But I still think your suggestion makes more sense and I have added it.